### PR TITLE
fix: Barcode readme

### PR DESCRIPTION
**Pull request by bensonarafat**

### CODE REVIEW

**Comment by fbernaly** [comment]
@bensonarafat: wait for this PR to be merged first: https://github.com/flutter-ml/google_ml_kit_flutter/pull/471

**Comment by bensonarafat** [comment]
okay

**Comment by fbernaly** [comment]
@bensonarafat : you can update your PR using the new API

File: packages/google_mlkit_barcode_scanning/README.md [hunk 1]
```diff
@@ -58,16 +58,16 @@ final List<Barcode> barcodes = await barcodeScanner.processImage(inputImage);
...
-  final String displayValue = barcode.displayValue;
-  final String rawValue = barcode.rawValue;
+  final String? displayValue = barcode.displayValue;
+  final String? rawValue = barcode.rawValue;
 
   // See API reference for complete list of supported types
   switch (type) {
     case BarcodeType.wifi:
-      BarcodeWifi barcodeWifi = barcode.value;
+      BarcodeValue? barcodeWifi = barcode.value;
```
Comment (fbernaly) [inline packages/google_mlkit_barcode_scanning/README.md:67]: change to 
`final barcodeWifi = barcode.value as BarcodeWifi;`

File: packages/google_mlkit_barcode_scanning/README.md [hunk 2]
```diff
@@ -58,16 +58,16 @@ final List<Barcode> barcodes = await barcodeScanner.processImage(inputImage);
...
       break;
-    case BarcodeValueType.url:
-      BarcodeUrl barcodeUrl = barcode.value;
+    case BarcodeType.url:
+      BarcodeValue? barcodeUrl = barcode.value;
```
Comment (fbernaly) [inline packages/google_mlkit_barcode_scanning/README.md:70]: change to
final barcodeUrl = barcode.value as BarcodeUrl;

File: packages/google_mlkit_barcode_scanning/README.md (hunk 1 above)
Comment (bensonarafat) [reply packages/google_mlkit_barcode_scanning/README.md:67]: okay

File: packages/google_mlkit_barcode_scanning/README.md (hunk 2 above)
Comment (bensonarafat) [reply packages/google_mlkit_barcode_scanning/README.md:70]: okay

**Comment by bensonarafat** [comment]
@fbernaly cool now

**Review by fbernaly — APPROVED** [review]

**Pull request merged**